src/main/java/com/carleton/CapstoneSystem/models/Program.java

```diff
@@ -1,7 +1,18 @@
 package com.carleton.CapstoneSystem.models;
 
 public enum Program {
-    SOFTWARE_ENGINEERING,COMPUTER_ENGINEERING,BIOMEDICAL_ENGINEERNG;
+    SOFTWARE_ENGINEERING("SE"),COMPUTER_ENGINEERING("CE"),BIOMEDICAL_ENGINEERNG("BE");

```

Comment:
Are there any other programs we should add?

Reply:
I agree, let's not be lazy. Please kindly add all the programs in this PR.

Reply:
I will add them can you please provide a list of what they are?

Reply:
you can check the programs we have from the last deploy.

Reply:
![image](https://user-images.githubusercontent.com/14824913/37673862-b470eb82-2c47-11e8-8214-9204c35349e7.png)